sph/test/kernel_ve/momentum_energy_kern.cpp

```diff
@@ -142,113 +140,9 @@ TEST(MomentumEnergy, JLoop)
                                          &du,
                                          &maxvsignal);
 
-    EXPECT_NEAR(grad_Px, -4.7423317200544224e-1, 1e-10);
-    EXPECT_NEAR(grad_Py, 8.737996639917453e-2, 1e-10);
-    EXPECT_NEAR(grad_Pz, -5.303834041425971e-1, 1e-10);
-    EXPECT_NEAR(du, -3.8485025262537881e-3, 1e-10);
```

Comment:
The reference changes here slightly, because the supplied input rho was not consistent with rho = kx * m / xmass.